ci-operator/step-registry/cucushift/ota/preupgrade/cucushift-ota-preupgrade-commands.sh

```diff
@@ -105,17 +116,26 @@ function run_ota_multi_test(){
 
 # Run single case through case ID
 function run_ota_single_case(){
-    if ! type pre-ocp-"${1}" &>/dev/null; then
-        echo "Test Failed: pre-ocp-${1} due to no case id found!" >> "${report_file}"
+    if ! type pre-"${1}" &>/dev/null; then
+        echo "WARN: no pre-${1} function found" >> "${report_file}"
     else
-        pre-ocp-"${1}"
-        if [[ $? == 1 ]]; then
-            echo "Test Failed: pre-ocp-${1}" >> "${report_file}"
+        echo "------> ${1}"
+        pre-"${1}"
+        if [[ $? == 0 ]]; then
+            echo "PASS: pre-${1}" >> "${report_file}"
+        else
+            echo "FAIL: pre-${1}" >> "${report_file}"
         fi
-    fi 
+    fi
 }
+
+if [[ "${ENABLE_OTA_TEST}" == "false" ]]; then
+  exit 0
+fi
+
 report_file="${ARTIFACT_DIR}/ota-test-result.txt"
 export PATH=/tmp:${PATH}
+extract_oc
```

Comment:
@jiajliu generally we do not need extract oc in pre action, do your case - 66839 need the latest oc version?